src/pyobs/aop.py

```diff
@@ -539,21 +731,23 @@ def Test_g2g_aop():
 
     # yaml.dump(rc,open('test.yml','w'))
 
-    data = '/Users/adasilva/data/'
-    aer_Nv = '/Users/adasilva/data/sampled/aer_Nv/CAMP2Ex-GEOS-MODISonly-aer-Nv-P3B_Model_*.nc'
-    
-    aer = xr.open_mfdataset(aer_Nv) # still having trouble with parallel
+    data = '/discover/nobackup/acollow/aeroeval/opticsfiles/AerosolOptics/'
+    #aer_Nv = '/Users/adasilva/data/sampled/aer_Nv/CAMP2Ex-GEOS-MODISonly-aer-Nv-P3B_Model_*.nc'
+    aer_Nv = '/discover/nobackup/acollow/CAMP2Ex/sampled/P3B/MODISonly/2019-09-*/CAMP2Ex-GEOS-MODISonly-aer-Nv-P3B_Model_201909*_R0.nc'
 
+    aer = xr.open_mfdataset(aer_Nv) # still having trouble with parallel
+    print('aer is loaded')
     g = G2GAOP(aer,mieRootDir=data,verbose=True)
-    rts = None # g.getAOPrt(wavelength=550,vector=False)
-    rtv = g.getAOPrt(wavelength=550,vector=True)
-    ext = None # g.getAOPext(wavelength=550)
-
-    return (g, rts, rtv, ext)
+    print('g is done')
+    #rts = None # g.getAOPrt(wavelength=550,vector=False)
```

Comment:
why did you take out the other examples. you should add yours to the already existing examples.